Google.Api.Gax.Grpc.GrpcNetClient/Google.Api.Gax.Grpc.GrpcNetClient.csproj

```diff
@@ -15,6 +15,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Google.Api.Gax.Grpc\Google.Api.Gax.Grpc.csproj" />
-    <PackageReference Include="Grpc.Net.Client" Version="[2.31.0, 3.0.0)"/>
+    <PackageReference Include="Grpc.Net.Client" Version="[2.36.0, 3.0.0)"/>
```

Comment:
Just to double check that this is righ. This one here is 2.36.0, the other two are 2.36.4 (patch version different)

Reply:
Yup, that's correct. Grpc.Net.Client doesn't have a 2.36.4. (That's the pure-C# implementation.)